recipes/guessmylt/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "guessmylt" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 30f9b300cc61c6f556605380bc4ba68d9eb3f3a5beb2f060cb69f9641f9c660c
+
+build:
+  number: 0
+  noarch: python
+  skip: True # [not win or osx]
+  entry_points:
+    - GUESSmyLT = GUESSmyLT.GUESSmyLT:main
+    - GUESSmyLT-example = GUESSmyLT.test.example:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - python>3
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
+    - pip
+    - pysam >=0.13.0
+    - snakemake ==5.4.*
+    - trinity ==2.8.4
+    - busco ==3.0.2
+    - bowtie2 >=2
+  run:
+    - python>3
+    - git >=2.11.1
+    - bcbio-gff ==0.6.4
+    - biopython ==1.67
```

Comment:
I'd still really like you to use `>=`.

As a general rule when your are publishing an application you should not pin exact versions, excepted when you know it breaks with the most recent.

Reply:
In another tool I had problem between bcbio-gff and biopython versions. I know this two versions work perfectly well together. I should do a test before accepting more recent versions.